Source/SIMPLib/DataContainers/AttributeMatrixProxy.cpp

```diff
@@ -285,3 +285,64 @@ void AttributeMatrixProxy::updatePath(DataArrayPath::RenameType renamePath)
     dataArrays.insert(newPath.getDataArrayName(), daProxy);
   }
 }
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+DataArrayProxy& AttributeMatrixProxy::getDataArrayProxy(const QString& name)
+{
+	for (QMap<QString, DataArrayProxy>::iterator iter = dataArrays.begin(); iter != dataArrays.end(); ++iter) // DataArray Level
```

Comment:
Check your indentation for this entire section.  Either you are using tabs or you are using a very different tab-to-spaces ratio here than everywhere else.

Reply:
There's also no newline at the end of the file, which we have everywhere else.  I'm not sure if we care about consistency for that, but apparently GitHub makes note of it.